plugin/src/test/kotlin/com/exactpro/th2/gradle/BaseTh2PluginTest.kt

```diff
@@ -29,17 +30,21 @@ import org.junit.jupiter.api.assertThrows
 import org.junit.jupiter.params.ParameterizedTest
 import org.junit.jupiter.params.provider.MethodSource
 import org.owasp.dependencycheck.gradle.DependencyCheckPlugin
+import org.owasp.dependencycheck.gradle.extension.AnalyzerExtension
+import org.owasp.dependencycheck.gradle.extension.DependencyCheckExtension
+import org.owasp.dependencycheck.gradle.extension.NvdExtension
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
 import kotlin.test.assertNotNull
+import kotlin.test.assertNull
 import kotlin.test.assertTrue
 
 @TestInstance(PER_CLASS)
 internal class BaseTh2PluginTest {
     @ParameterizedTest(name = "when {0} plugin applied")
     @MethodSource("requiredPlugins")
     fun `applies required plugins`(javaPlugin: String) {
-        val project =
-            ProjectBuilder.builder()
-                .build()
+        val project = ProjectBuilder.builder().build()
```

Comment:
question: Why do you need to change the formatting in places where you don't change the code?

Reply:
I saw warning in idea